challenges/2022-10-11-blast-from-the-past/setups/rust/Cargo.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+name = "rust"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
```

Comment:
Is this required even with nothing defined?

Reply:
Nah, it's just auto-generated from cargo.